src/Homework3/task7_3.py

```diff
@@ -0,0 +1,57 @@
+"""Шоколадка. на подумать если есть желание. точного решения у меня нет**
+   Определения:
+   - Шоколадка - прямоугольник, размером n×m (n, m - натуральные).
+   - Разлом - деление шоколадки на две части с натуральными размерами по
+   прямой.
+   - Долька - элемент шоколадки размером 1х1. Очевидно шоколадка состоит
+   из n*m долек.
+   - Кусок - элемент шоколадки произвольного (целочисленного размера).
+  Определите:
+   1) можно ли одним разломом отделить от шоколадки кусок площадью ровно k.
+   2) можно ли отломить от шоколадки ровно k долек за некоторое количество
+   разломов.
+   3) можно ли отломить от шоколадки ровно k долек с помощью t разломов
+"""
+
+# Это решение работает, если k долек следует отломить минимальным количеством
```

Comment:
we should split k parts with t cuts. it should be entered by user (can be any number, not minimal).